ui/admin/app/templates/scopes/scope/credential-stores/credential-store/credentials/index.hbs

```diff
@@ -28,18 +28,18 @@
   <page.body>
     {{#if @model}}
 
-      <Rose::Table as |table|>
-        <table.header as |header|>
-          <header.row as |row|>
-            <row.headerCell>{{t 'form.name.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.type.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.id.label'}}</row.headerCell>
-          </header.row>
-        </table.header>
-        <table.body as |body|>
+      <Hds::Table>
+        <:head as |H|>
```

Comment:
Why we are not using table attributes `@model` and `@columns` as we did with the other HDS::Table migration?

Reply:
@calcaide the model for this one is returned as a proxy array  and therefore does not work when providing that model to the table, any suggestions?

Reply:
@dvonanderson 
I don't see why it does not work. What problem do you encounter when trying so?

Reply:
@calcaide this is really weird, because before holiday break whenever I tried to use the `.data` key to access the `@model` record content, I would get a error message stating that I was using the incorrect data type.  (should have recorded that message and reached out then). I looked at how the model was being returned and it was using a `proxy-array`.  Not really sure of the dynamics of a `proxy-array` but it seems to be working perfectly fine now.  

<img width="1691" alt="Screenshot 2024-01-02 at 09 49 45" src="https://github.com/hashicorp/boundary-ui/assets/24277002/d5ca302c-a7c8-4a10-bf8e-585e66824dac">

Reply:
I am not sure what value are you logging on the console, but as stated above, I don't see why an `ArrayProxy` would be a problem here.
Perhaps, before anything, we need to understand the dynamics of it, [here documentation](https://api.emberjs.com/ember/4.12/classes/ArrayProxy).